sources/packages/api/src/cra-integration/cra-files/cra-response-status-record.ts

```diff
@@ -41,6 +45,38 @@ export class CRAResponseStatusRecord extends CRAResponseRecordIdentification {
     return this.line.substr(29, 2) as MatchStatusCodes;
   }
 
+  /**
+   * SIN match status codes (SIN-TTN-STATUS-CODE) presents on
+   * CRA Response Record (Trans Sub Code - 0022).
+   */
+  public get sinMatchStatusCode(): SINMatchStatusCodes {
+    return this.line.substr(31, 2) as SINMatchStatusCodes;
```

Comment:
`substr` is depreciated, and we have started using `substring`
![image](https://user-images.githubusercontent.com/77353155/157523999-48672c8e-6c70-4bea-8fc5-39f0d00fcdda.png)
![image](https://user-images.githubusercontent.com/77353155/157524277-2350269c-0e5a-4555-b484-557e5f576ca2.png)